src/riak_ql_window_agg_fns.erl

```diff
@@ -61,6 +66,11 @@ fn_type_signature('SUM', [timestamp]) -> timestamp;
 fn_type_signature(Fn, Args) ->
     {error, {argument_type_mismatch, Fn, Args}}.
 
+-spec fn_param_check(aggregate_function(), [riak_ql_ddl:external_field_type()]) ->
```

Comment:
I understand a bit better now - you are calling `Mod:fn_param_check` and `windows_agg_fns` don't have a param to check - but won't this cause a dialzer error?